contracts/L2ForkArbitrator.sol

```diff
@@ -75,6 +56,8 @@ contract L2ForkArbitrator is IBridgeMessageReceiver {
     );
 
     mapping(bytes32 => ArbitrationRequest) public arbitrationRequests;
```

Comment:
maybe I can unit these two datatypes

Reply:
I decided against it, since the inputs are provided in different situations and hence a uniation is not necessary